test/integration/ci-operator-config-mirror.sh

```diff
@@ -10,7 +10,7 @@ expected="${suite_dir}/output"
 os::test::junit::declare_suite_start "integration/ci-operator-config-mirror"
 # This test validates the ci-operator-config-mirror tool
 
-os::cmd::expect_success "ci-operator-config-mirror --to-org super-priv --config-path ${actual} --whitelist-file ${suite_dir}/whitelist.yaml"
+os::cmd::expect_success "ci-operator-config-mirror --to-org super-priv --config-path ${actual} --deprecate-configs=false --whitelist-file ${suite_dir}/whitelist.yaml"
```

Comment:
We should have a test with `--deprecated-configs=true`, too (as that is the configuration we will want to run).